src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ApiAccessSteps.java

```diff
@@ -62,4 +62,24 @@ public void validateResponseCP(final String dbId, final String chplId) {
         Assert.assertEquals(chplProductNumberInDetails, chplId, "CHPL ID is as expected");
         Assert.assertEquals(chplProductNumberInBasic, chplProductNumberInDetails, "CHPL IDs match as expected");
     }
+
+    /**
+     * Runs API call request using given end point to get cache status in response.
+     * @param status is the expected status in response
+     */
+    @Then("^the cache_status endpoint returns status \"([^\"]*)\" for all caches are populated status$")
+    public void validateResponseCacheStatus(final String status) {
+
+        Response response = given()
+                .header("API-KEY", apikey)
```

Comment:
You don't need an API-Key to access the /status endpoints. It's fine to have it, but not required. That said, I think the header is "API-Key", not "API-KEY", though I could be mistaken